### PR TITLE
Making the footer in VIVO look like the footer in Drupal

### PR DESCRIPTION
**Pull request by mrpatrickwest**

### CODE REVIEW

**Comment by mrpatrickwest** [comment]
on udco at http://udco.tw.rpi.edu/vivo/

File: themes/dco/templates/footer.ftl [hunk 1]
```diff
@@ -29,6 +30,39 @@
...
+    -->
+    <table style="background-color: black;">
+	<tr>
+	    <td style="width:15%;padding:1em;">
+<ul style="list-style-type:circle;">
+<li class="boiler"><a class="dco_anchor" href="/vivo/" title="">Home</a></li>
+<li class="boiler"><a class="dco_anchor" target="_blank" href="https://deepcarbon.net/about/about-dco" title="">About</a></li>
+<li class="boiler"><a class="dco_anchor" target="_blank" href="https://deepcarbon.net" title="">Community Portal</a></li>
+<li class="boiler"><a class="dco_anchor" href="/vivo/" title="">Data Portal</a></li>
+<li class="boiler"><a class="dco_anchor" target="_blank" href="https://deepcarbon.net/content/news" title="">News</a></li>
```
Comment (zednis) [inline themes/dco/templates/footer.ftl:42]: Should we use relative URLs here for all the links?

Comment (mrpatrickwest) [inline themes/dco/templates/footer.ftl:42]: VIVO is https://info.deepcarbon.net so the https://deepcarbon.net (in Drupal) links wouldn't be relative, and is also why I open them in a new tab/window.

Comment (zednis) [inline themes/dco/templates/footer.ftl:42]: So these are links to the production drupal site then?

edit - nm, I see you answered my question in the previous post.

**Comment by mrpatrickwest** [comment]
@zednis are we good to go with this?

**Comment by zednis** [comment]
:shipit:

**Pull request merged**